platform-api/src/test/java/io/datacater/core/info/InfoEndpointTest.java

```diff
@@ -0,0 +1,18 @@
+package io.datacater.core.info;
+
+import static io.restassured.RestAssured.given;
+
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@QuarkusTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestHTTPEndpoint(InfoEndpoint.class)
+public class InfoEndpointTest {
+  @Test
+  void testGetInfo() {
+    given().get().then().statusCode(200);
+  }
```

Comment:
Are we able to provide a few more test cases, which test the response body?

Reply:
I added tests to check everything except for the version info, since that is subject to change pretty often and most of the values are empty/default for testing/dev environments.